src/main/java/org/apache/parquet/hadoop/BdecParquetWriter.java

```diff
@@ -43,6 +43,10 @@ public class BdecParquetWriter implements AutoCloseable {
   // Optional cap on the max number of row groups to allow per file, if this is exceeded we'll end
```

Comment:
nit: lets rename this file to SnowflakeParquetWriter instead of BdecParquetWriter?!